docs/_index.md

```diff
@@ -38,30 +38,28 @@ While `super` and its accompanying data formats are production quality, the proj
 a number of different elements of the system:
 * The [super data model](formats/zed.md) is the abstract definition of the data types and semantics
 that underlie the super-structured data formats.
-* The [super data formats](formats/_index.md) are a family of
+* The [super-structured data formats](formats/_index.md) are a family of
 [human-readable (Super JSON, JSUP)](formats/jsup.md),
 [sequential (Super Binary, BSUP)](formats/bsup.md), and
 [columnar (Super Columnar, CSUP)](formats/csup.md) formats that all adhere to the
 same abstract super data model.
-* The [SuperPipe language](language/_index.md) is the system's pipeline language for performing
+* [SuperSQL](language/_index.md) is the system's language for performing
 queries, searches, analytics, transformations, or any of the above combined together.
-* A  [SuperPipe query](language/overview.md) is a script that performs
-search and/or analytics.
-* A [SuperPipe shaper](language/shaping.md) is a script that performs
-data transformation to _shape_
-the input data into the desired set of organizing super-structured data types called "shapes",
-which are traditionally called _schemas_ in relational systems but are
-much more flexible in SuperDB.
+* A [SuperSQL pipe query (SPQ)](language/overview.md) is a query that
+employs SuperSQL's unique pipeline extensions and shortcuts to perform data
+operations that are difficult or impossible in standard SQL.
 * A [SuperDB data lake](commands/super-db.md) is a collection of super-structured data stored
 across one or more [data pools](commands/super-db.md#data-pools) with ACID commit semantics and
 accessed via a [Git](https://git-scm.com/)-like API.
 
 ## Digging Deeper
 
-The [SuperPipe language documentation](language/_index.md)
-is the best way to learn about `super` in depth.
-All of its examples use `super` commands run on the command line.
-Run `super -h` for a list of command options and online help.
+The [SuperSQL language documentation](language/_index.md)
+is the best way to learn about `super` in depth. Most
+[examples](commands/super.md#examples) that appear throughout the docs can be
+executed right in your browser and can easily be copied to the command line
+for execution with `super`. Run `super -h` for a list of command options and
+brief help.
```

Comment:
This revision is largely a nod to the new Playground presentation that also includes the CLI tab. See also #5627.